test/unit/conftest.py

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
This is partially covered in the PR description. I'll extend it with more cases.

Reply:
It's important to have the examples of how to run it in test/unit/README.md, test/unit/conftest.py, and/or perhaps test/README.md and docs/dev/testing.md.
Because nobody will ever see your commit message the day after it's merged, and it's hard to guess how to run these tests (do you use the name of the compiled test or the source to run one test? and so on).